docs/migrations/v3-to-v4.md

```diff
@@ -18,13 +18,49 @@ No genesis or in-place migrations required when upgrading from v1 or v2 of ibc-g
 
 ## Chains
 
-### IS04 - Channel 
+### IS04 - Channel
 
-The `WriteAcknowledgement` API now takes the `exported.Acknowledgement` type instead of passing in the acknowledgement byte array directly. 
-This is an API breaking change and as such IBC application developers will have to update any calls to `WriteAcknowledgement`. 
+The `WriteAcknowledgement` API now takes the `exported.Acknowledgement` type instead of passing in the acknowledgement byte array directly.
+This is an API breaking change and as such IBC application developers will have to update any calls to `WriteAcknowledgement`.
 
 ## IBC Light Clients
 
-The `VerifyUpgradeAndUpdateState` function has been modified. The client state and consensus state return value has been removed. 
+### ClientState interface changes
 
-Light clients **must** set the updated client state and consensus state in the client store after verifying a valid client upgrade.
+The `VerifyUpgradeAndUpdateState` function has been modified. The client state and consensus state return value has been removed.
```

Comment:
```suggestion
The `VerifyUpgradeAndUpdateState` function has been modified. The client state and consensus state return values has been removed.
```

Reply:
Yeah, but I think it should be like this:
"The `VerifyUpgradeAndUpdateState` function has been modified. The returned client state and consensus state values has been removed."